pkg/genericapiserver/genericapiserver.go

```diff
@@ -627,6 +630,11 @@ func (s *GenericAPIServer) Run(options *ServerRunOptions) {
 	glog.Fatal(http.ListenAndServe())
 }
 
+func fileExists(path string) bool {
+	_, err := os.Stat(path)
+	return !os.IsNotExist(err)
```

Comment:
@lavalamp actually I just want to make sure this file exists here, if we get a permission error, of course the file exists. Guess you want to make sure this file exists and also readable(we have permission), I will update this to make it more robust and generic.